app/models/social_networking/reports/like.rb

```diff
@@ -15,8 +15,9 @@ def self.all
 
               {
                 participant_id: participant.study_id,
-                occurred_at: like.created_at.iso8601,
-                item_type: item.class.to_s
+                created_at: like.created_at,
```

Comment:
Should I be concerned that above it in the ```self.columns```, it says ```occurred_at``` instead of ```created_at``` or is the terminology "occurred_at"?  Thanks!